### PR TITLE
Removing ReceiveDisabled status from EventHub and ServiceBus CLI doc as it's an invalid status

### PR DESCRIPTION
**Pull request by basilhariri**
ReceiveDisabled is a valid status for consumer groups, but not for EventHubs or topics/queues. Trying to set any of those entities to ReceiveDisabled will cause an error.

=====

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@Juliehzl please help with this PR.

File: src/azure-cli/azure/cli/command_modules/eventhubs/_params.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def load_arguments_eh(self, _):
         with self.argument_context(scope) as c:
             c.argument('message_retention_in_days', options_list=['--message-retention'], type=int, help='Number of days to retain events for this Event Hub, value should be 1 to 7 days and depends on Namespace sku. if Namespace sku is Basic than value should be one and is Manadatory parameter. Namespace sku is standard value should be 1 to 7 days, default is 7 days and is optional parameter')
             c.argument('partition_count', type=int, help='Number of partitions created for the Event Hub. By default, allowed values are 2-32. Lower value of 1 is supported with Kafka enabled namespaces. In presence of a custom quota, the upper limit will match the upper limit of the quota.')
-            c.argument('status', arg_type=get_enum_type(['Active', 'Disabled', 'SendDisabled', 'ReceiveDisabled']), help='Status of Eventhub')
+            c.argument('status', arg_type=get_enum_type(['Active', 'Disabled', 'SendDisabled']), help='Status of Eventhub')
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/eventhubs/_params.py]: Removing an enum type will be a breaking change. Given no many usages for the parameter, it is acceptable but please declare this breaking change in history.rst with [BREAKING CHANGE].

Comment (basilhariri) [reply src/azure-cli/azure/cli/command_modules/eventhubs/_params.py]: Fixed!

**Comment by Juliehzl** [comment]
Hi @basilhariri, please resolve the conflicts and please make your release notes in next release part in history.rst, which should be above 2.0.78.

**Comment by basilhariri** [comment]
Hi @Juliehzl and @yonzhan, it seems like this has been approved, but not merged. Do we need someone else's help to merge it into dev?

**Pull request merged**